.gitlab-ci.yml

```diff
@@ -6,6 +6,8 @@ variables:
   # search for and change them too.
   MENDER_ARTIFACT_VERSION: master
   MENDER_CLIENT_VERSION: latest
+  # MENDER_ADDON_CONNECT_VERSION: latest
```

Comment:
?

Reply:
It is not yet published in stable. See https://github.com/mendersoftware/mender-convert/blob/master/configs/mender_convert_config#L131 and https://tracker.mender.io/browse/MEN-4200